test/units/plugins/action/test_action.py

```diff
@@ -627,6 +627,6 @@ def test_action_base_sudo_only_if_user_differs(self):
         try:
             play_context.remote_user = 'root'
             action_base._low_level_execute_command('ECHO SAME', sudoable=True)
-            play_context.make_become_cmd.assert_called_once_with("ECHO SAME", executable='/bin/sh')
+            play_context.make_become_cmd.assert_called_once_with("ECHO SAME", executable=None)
```

Comment:
defaults to self.executable, which gets set from inventory or if all else fails the old DEFAULT_EXECUTABLE, which is `/bin/sh` by default